src/main/java/com/uid2/admin/vertx/service/EncryptionKeyService.java

```diff
@@ -68,8 +74,11 @@ public EncryptionKeyService(JsonObject config,
 
         masterKeyActivatesIn = Duration.ofSeconds(config.getInteger(MASTER_KEY_ACTIVATES_IN_SECONDS));
         masterKeyExpiresAfter = Duration.ofSeconds(config.getInteger(MASTER_KEY_EXPIRES_AFTER_SECONDS));
+        masterKeyRotationCutoffTime = Duration.ofDays(config.getInteger(MASTER_KEY_ROTATION_CUT_OFF_DAYS));
```

Comment:
What happens if these are not specified in config? Does it return 0 or throw an exception? If it does return 0, what will the code do?

Reply:
to be safe better set a huge default value in code if it can't be read from config

Reply:
you can use getInteger(String key, Integer def) method to set a default value if config didn't specify a value